cosmos/operators/local.py

```diff
@@ -59,6 +60,13 @@
         logger.exception(error)
         is_openlineage_available = False
 
+        @define
+        class OperatorLineage:  # type: ignore
+            inputs: list[str] = list()
+            outputs: list[str] = list()
+            run_facets: dict[str, str] = dict()
+            job_facets: dict[str, str] = dict()
```

Comment:
Nit: if it's just for typing, then maybe just hide the import behind `TYPE_CHECKING`?

Reply:
I'll address this in a separate PR :)